src/NServiceBus.Persistence.CosmosDB/CosmosPersistence.cs

```diff
@@ -7,9 +7,9 @@
     /// <summary>
     /// CosmosDB Core API persistence
```

Comment:
```suggestion
    /// Cosmos DB Core API persistence
```

Reply:
Should be addressed in https://github.com/Particular/NServiceBus.Persistence.CosmosDB/pull/67